sklearn/metrics/tests/test_ranking.py

```diff
@@ -355,6 +354,24 @@ def test_roc_curve_toydata():
     assert_array_almost_equal(fpr, [0, 1])
     assert_almost_equal(roc_auc, 0.5)
 
+    y_true = [1, 1]
```

Comment:
Do we really need to change the order here? I think in main this was below the case where y_true = [0, 0] and y_score = [0.25, 0.75]. So this difference seems extra.